### PR TITLE
Add integration guide for Managed Updates

### PR DESCRIPTION
**Pull request by huyhg**

### CODE REVIEW

File: docs/managed-updates.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+
+See [this doc](tool-prerequisites.md).
+
+### GCS bucket
+
+Create a GCS bucket to house the published version metadata. This SHOULD
+be a bucket in the same project you're publishing images and SHOULD be
+dedicated to this process.
+
+The bucket MUST be set to be publicly _readable and listable_. See the
```
Comment (vcanaa) [inline docs/managed-updates.md]: Is this a typo?
_readable
listable_

File: docs/managed-updates.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+
+```shell
+
+mpdev publish \
+  --deployer_image=gcr.io/your-project/your-company/your-app/deployer:1.0.1 \
+  --gcs_repo=gs://your-bucket/your-company/your-app/1.0
+```
+
+This should print out the URL of the published metadata file. Its location
+can be determined deterministically from the GCS repo and the version.
```
Comment (vcanaa) [inline docs/managed-updates.md]: Its location is deterministically derived from the GCS repo and the version.

File: docs/managed-updates.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+and name prefix rule. For example, the published images could be under:
+- `marketplace.gcr.io/your-company/your-app:1.0.1`
+- `marketplace.gcr.io/your-company/your-app/deployer:1.0.1`
+- `marketplace.gcr.io/your-company/your-app/init:1.0.1`
+
+### Properties and Required
+
+Properties and required sections remain largely the same.
+Note that because images are now declared in the top level `x-google-marketplace`
+section, there MUSTN'T be any properties with `.x-google-marketplace.type=IMAGE`.
```
Comment (vcanaa) [inline docs/managed-updates.md]: I forgot, do we validate this?

File: docs/managed-updates.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+  - `gs://your-bucket/your-company/your-app/1.0/1.0.2.yaml`
+
+You do not have to craft these files directly. `mpdev publish` takes care of
+this. See below.
+
+Note that end users consuming your application via the public Marketplace
+will pull the version metadata from the Marketplace's public GCS bucket instead
+of the one you use during development, i.e. `your-bucket`.
+
+p## Image organization
```
Comment (wgrzelak) [inline docs/managed-updates.md]:
```suggestion
## Image organization
```

File: docs/managed-updates.md (hunk 1 above)
Comment (huyhg) [reply docs/managed-updates.md]: Revised this paragraph to include a direct CLI command instead.

File: docs/managed-updates.md (hunk 3 above)
Comment (huyhg) [reply docs/managed-updates.md]: Filed a bug to track this internally.

**Pull request merged**